testng-junit-migrator/src/main/java/tech/picnic/errorprone/testngjunit/TimeOutAttributeMigrator.java

```diff
@@ -0,0 +1,37 @@
+package tech.picnic.errorprone.testngjunit;
+
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.annotations.Immutable;
+import com.google.errorprone.fixes.SuggestedFix;
+import com.sun.source.tree.MethodTree;
+import java.util.Optional;
+import tech.picnic.errorprone.util.SourceCode;
+
+/**
+ * A {@link AttributeMigrator} that migrates the {@code org.testng.annotations.Test#timeOut}
+ * attribute. The TestNG {@code org.testng.annotations.Test#timeOut} attribute is always in
+ * milliseconds, the JUnit variant {@code @Timeout} takes a value in seconds by default, hence we
+ * add the {@link java.util.concurrent.TimeUnit#MILLISECONDS} attribute.
+ */
+@Immutable
+final class TimeOutAttributeMigrator implements AttributeMigrator {
+  @Override
+  public Optional<SuggestedFix> migrate(
+      TestNGMetadata metadata,
+      TestNGMetadata.AnnotationMetadata annotation,
```

Comment:
If we rename `AnnotationMetaData` to `TestNGAnnotationMetdata` we can omit the `TestNGMetadata` qualifier everywhere, I think that is cleaner, WDYT?

Reply:
So iiuc, we extract the `AnnotationMetadata` to a seperate class and call it `TestNGAnnotationMetadata`?